test/models/company_test.exs

```diff
@@ -3,19 +3,34 @@ defmodule Appointments.CompanyTest do
 
   alias Appointments.Company
 
-  @valid_attrs %{description: "some content",
-                 email: "some content",
-                 name: "some content",
-                 phone: "some content"}
-  @invalid_attrs %{}
-
-  test "changeset with valid attributes" do
-    changeset = Company.changeset(%Company{}, @valid_attrs)
+  test "validations: website_url should be comply to the URI standard" do
+    changeset = Company.changeset(%Company{}, %{name: "a", website_url: "mbl"})
+    refute changeset.valid?
+
+    changeset = Company.changeset(
+      %Company{}, %{name: "a", website_url: "mbl.is"})
+    refute changeset.valid?
+
+    changeset = Company.changeset(
+      %Company{}, %{name: "a", website_url: "http://mbl.is"})
+    assert changeset.valid?
+  end
+
+  test "validations: location_country should be a ISO alpha2" do
+    changeset = Company.changeset(%Company{}, %{name: "A"})
+    assert changeset.valid?
```

Comment:
You're right this looks stupid. This is actually asserting that `location_country` can be `nil` but that should belong in another test with a proper description.